gui/inputs/confidence_level.py

```diff
@@ -0,0 +1,28 @@
+from tkinter.ttk import Label, OptionMenu
+
+from gui.inputs.abstract.levels import Levels
+
+"""
+This class is similar to ThresholdLevel 
+it will accept a container in the constructor 
+to render the widgets.
+
+
+"""
+
+
+class ConfidenceLevel(Levels):
+    def __init__(self, parent_container):
+        super().__init__(parent_container)
+
+    def render_widgets(self, r):
+        Label(self.container, text="Min confidence % : ") \
+            .grid(sticky='w', row=r, column=1)
+
+        OptionMenu(self.container, self.level,
+                   command=self.set_new_level,
+                   *range(50, 101, 10))\
```

Comment:
optionmenu may have a bug where it doesnt show the minimum option (from range) in the dropdown. weird.